pkg/html/generichtml/jobaggregationresult.go

```diff
@@ -137,95 +138,110 @@ func (b *jobAggregationResultRenderBuilder) StartCollapsedAs(collapsedAs string)
 	s := ""
 
-	// TODO either make this a template or make this a builder that takes args and then has branches.
-	//  that will fix the funny link that goes nowhere.
-	template := `
-			<tr class="%s">
-				<td>
-					%s
-					<p>
-					%s
-				</td>
-				<td>
-					%0.2f%% (%0.2f%%)<span class="text-nowrap">(%d runs)</span>
-				</td>
-				<td>
-					%s
-				</td>
-				<td>
-					%0.2f%% (%0.2f%%)<span class="text-nowrap">(%d runs)</span>
-				</td>
-			</tr>
-		`
-
-	naTemplate := `
-			<tr class="%s">
-				<td>
-					%s
-					<p>
-					%s
-				</td>
-				<td>
-					%0.2f%% (%0.2f%%)<span class="text-nowrap">(%d runs)</span>
-				</td>
-				<td/>
-				<td>
-					NA
-				</td>
-			</tr>
-		`
-
-	class := b.colors.GetColor(b.currAggregationResult.displayPercentage, b.currAggregationResult.totalJobRuns)
-	if len(b.collapsedAs) > 0 {
-		class += " collapse " + b.collapsedAs
-	}
+	buttons := ""
+	jobsCollapseName := b.sectionBlock
 
 	prevTestResults := []testResultDisplay{}
 	if b.prevAggregationResult != nil {
 		prevTestResults = b.prevAggregationResult.testResults
 	}
 
 	testCollapseSectionName := MakeSafeForCollapseName(b.sectionBlock + "---" + b.currAggregationResult.displayName + "---tests")
-	jobsCollapseName := MakeSafeForCollapseName(b.sectionBlock + "---" + b.currAggregationResult.displayName + "---jobs")
 	testRows, displayedTests := getTestRowHTML(b.release, testCollapseSectionName, b.currAggregationResult.testResults, prevTestResults, b.maxTestResultsToShow)
-	button := "					" + GetExpandingButtonHTML(jobsCollapseName, "Expand Failing Jobs")
+
 	if len(displayedTests) > 0 { // add the button if we have tests to show
-		button += " " + GetExpandingButtonHTML(testCollapseSectionName, "Expand Failing Tests")
-		button += " " + GetTestDetailsButtonHTML(b.release, displayedTests...)
+		buttons += " " + GetExpandingButtonHTML(testCollapseSectionName, "Expand Failing Tests")
 	}
 
-	if b.prevAggregationResult != nil {
-		arrow := GetArrow(b.currAggregationResult.totalJobRuns, b.currAggregationResult.displayPercentage, b.prevAggregationResult.displayPercentage)
-
-		s = s + fmt.Sprintf(template,
-			class,
-			b.currAggregationResult.displayName,
-			button,
-			b.currAggregationResult.displayPercentage,
-			b.currAggregationResult.parenDisplayPercentage,
-			b.currAggregationResult.totalJobRuns,
-			arrow,
-			b.prevAggregationResult.displayPercentage,
-			b.prevAggregationResult.parenDisplayPercentage,
-			b.prevAggregationResult.totalJobRuns,
-		)
-	} else {
-		s = s + fmt.Sprintf(naTemplate,
-			class,
-			b.currAggregationResult.displayName,
-			button,
-			b.currAggregationResult.displayPercentage,
-			b.currAggregationResult.parenDisplayPercentage,
-			b.currAggregationResult.totalJobRuns,
-		)
+	// TODO either make this a template or make this a builder that takes args and then has branches.
+	// that will fix the funny link that goes nowhere.
+	if b.groupby == "variant" {
```

Comment:
When we aren't displaying a variant, what does this show?  Are you using this just to create a table of the jobs?

Reply:
Job aggregation result is what the main page uses, I wanted to reuse that same code. This if hides the variant header. Based on your comments, I guess I'm missing something.